lmwrapper/huggingface_wrapper.py

```
@@ -313,20 +315,28 @@ def get_huggingface_lm(
     runtime: Runtime = Runtime.PYTORCH,
     precision: torch.dtype = torch.float32,
 ) -> HuggingfacePredictor:
-    _kwargs = {}
+    _kwargs = {"trust_remote_code": True}
```

this feels like a dubious default. If huggingface/whoever thinks it should not be a default, not sure we want to expose users of our wrapper to these risks (the audience here is people who don't actually want to deal with all the intricacies of HF and just want reasonable defaults). This might need to be as a user specified flag and ideally caught with an informative error message.

---

Agreed. I will work on adding AutoConfig loading, and checking the config whether it specifies remote classes. If a remote class is specified, I think lmwrapper should raise an exception if `trust_remote_code` is False (as it should be). The problem here is that many new models have custom model classes on HF Hub, and the default behavior leads to unexpected results as the correct model class may not be loaded.